tests/testprofiles/tests.py

```diff
@@ -559,3 +561,34 @@ def test_user_cleaned_main_attribute(self):
 
         self.user.refresh_from_db()
         self.assertEqual(user.username, "john")
+
+
+class CSPHandlerTests(TestCase):
+    def test_get_csp_handler_none(self):
+        get_csp_handler.cache_clear()
+        with override_settings(SAML_CSP_HANDLER=None):
+            csp_handler = get_csp_handler()
+            assert (
+                csp_handler.__name__ == "empty_view_decorator"
+                or csp_handler.__module__ == "csp.decorators"
+            )
+
+    def test_get_csp_handler_empty(self):
+        get_csp_handler.cache_clear()
+        with override_settings(SAML_CSP_HANDLER=""):
+            csp_handler = get_csp_handler()
+            assert csp_handler.__name__ == "empty_view_decorator"
```

Comment:
we should not use `assert`, using TestCase we should use self.assertEqual ... self.assertTrue, and so on

Reply:
:+1: I haven't used django's builtin test stuff in many years so I default to the pytest approaches.